.github/workflows/release.yml

```
@@ -5,8 +5,8 @@
 name: Подготовка релиза и публикация в хабе
 # Только события создания и изменения релиза
 on:
-    release:
-        types: [published, edited]
+  release:
+      types: [published]
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)